test/verify/check-users

```diff
@@ -248,7 +248,6 @@ class TestAccounts(MachineCase):
         b.wait_visible("#account-set-password-meter.danger")
         b.click('#account-set-password-dialog button.apply')
         b.wait_in_text("#account-set-password-dialog .pf-c-form__helper-text.pf-m-error", "Password quality check failed:")
-        b.wait_in_text("#account-set-password-dialog .pf-c-form__helper-text.pf-m-error", "Click Set password again to use the password anyway.")
```

Comment:
Here as well please.

Reply:
And on line 257 you could wait for the button to not appear when `passwords do not match`.